MINT.postman_collection.json

```diff
@@ -3058,7 +3058,7 @@
 								"body": {
 									"mode": "graphql",
 									"graphql": {
-										"query": "mutation NewMTOMilestone {\ncreateMTOMilestone(modelPlanID: \"{{modelPlanID}}\",\n# name: \"Uncategorized commonMilestone \"\ncommonMilestoneKey: MILESTONE_A\n# ,mtoCategoryID: \"{{mtoCategoryID}}\"\n# ,mtoCategoryID: \"{{mtoSubcategoryID}}\"\n) {\n    id\n    name\n    key\n    addedFromMilestoneLibrary\n    facilitatedBy\n    needBy\n    status\n    riskIndicator\n    # isDraftMilestone\n    mtoCategoryID\n\n    # commonMilestone\n    # solutions\n    # category\n    # subCategory\n    }\n}",
+										"query": "mutation NewMTOMilestone {\n    createMTOMilestoneCommon(modelPlanID: \"{{modelPlanID}}\",\n    commonMilestoneKey: MILESTONE_A\n) {\n    id\n    name\n    key\n    addedFromMilestoneLibrary\n    facilitatedBy\n    needBy\n    status\n    riskIndicator\n    # isDraftMilestone\n    mtoCategoryID\n\n    # commonMilestone\n    # solutions\n    # category\n    # subCategory\n    }\n}",
 										"variables": ""
```

Comment:
Same as above, the script tab needs to have the script updated to reference `createMTOMilestoneCommon`

Reply:
d98f42148